JokeRandomizer/JokeRandomizer/ViewModel/JokesViewModel.swift

```diff
@@ -7,46 +7,54 @@
 
 
 import Foundation
+import Combine
 
-class JokesViewModel: ObservableObject {
+class JokesViewModel {
+    private var cancellables = Set<AnyCancellable>()
     
-    private(set) var joke: [Jokes] = []
-    
-    func getJokesVM(url: URL) async {
-        do {
-        let jokes = try await WebService().getJokes(url: url)
-            self.joke = jokes
-        } catch {
-            print(error)
-        }
+    enum Input {
+        case viewDidAppear
+        case refreshButtonTap
     }
 
+    enum Output {
```

Comment:
Combine is great but use closures for now instead.

Reply:
Still working on how to move it on the closure.